fixtures.http/api/getDomainPrices/success.http

```diff
@@ -18,4 +18,4 @@ X-Permitted-Cross-Domain-Policies: none
 Content-Security-Policy: frame-ancestors 'none'
 Strict-Transport-Security: max-age=31536000
 
-{"data":{"domain":"bingo.pizza","premium":true,"registration_price":20.0,"renewal_price":20.0,"transfer_price":20.0}}
+{"data":{"domain":"bingo.pizza","premium":true,"registration_price":20.0,"renewal_price":20.0,"restore_price":15.0,"transfer_price":20.0}}

```

Comment:
How did you generate this fixture? The fixtures in this folder should be the ones at https://github.com/dnsimple/dnsimple-developer/, but I see https://github.com/dnsimple/dnsimple-developer/commit/712d2e6dae30de54786ccf725eb1d84f1359201d is different than this one.

Reply:
@weppos I manually modified this fixture to add in the restore price by referencing the response shown on the developer site. I tweaked the restore price value so that it would be different from the other values, but I can make the fixtures an exact match if that's better

Reply:
> @weppos I manually modified this fixture to add in the restore price by referencing the response shown on the developer site. I tweaked the restore price value so that it would be different from the other values, but I can make the fixtures an exact match if that's better

The source of truth are the fixtures in the developer site. The ones here, are supposed to be copies of that one. At any time, we should be able to delete the folder in this repo, and replace it with the one from the developer site.

Therefore, if you want any change here, you need to apply it on the developer site then pull the changes here.

Reply:
@weppos Thanks for the explanation! I have made the fixture in this repository exactly the same as the one from the developer site: https://github.com/dnsimple/dnsimple-go/pull/198/commits/e5df6ea2b6e39dc680c409708d67cb0b3a40a75c